src/views/Dashboard.vue

```diff
@@ -246,8 +246,10 @@ export default {
     )
     this.subscribe('root')
   },
-  beforeDestroy () {
+  beforeRouteLeave (to, from, next) {
```

Comment:
This is necessary, as the View is not just a component, but bound to a Vue Router router. So the lifecycle is different. Views don't get destroyed as Vue components do. So the `beforeDestroy` doesn't always get called.